{"name": "test_add_contact_to_group", "status": "broken", "statusDetails": {"message": "pony.orm.core.UnrepeatableReadError: Phantom object ORMContact[59] disappeared", "trace": "app = <fixture.application.Application object at 0x7fd92c83d2e0>, orm = <fixture.orm.ORMFixture object at 0x7fd92c72d880>\n\n    def test_add_contact_to_group(app, orm):\n        if len(app.contact.get_contact_list()) == 0:\n            app.contact.create_new_contact(NewContact(first_name=\"Red21\", last_name=\"Gena\"))\n        if len(app.group.get_group_list()) == 0:\n            app.group.create(Group(name=\"name33\", header=\"headre23\"))\n        all_contacts = orm.get_contact_list()\n        contact = random.choice(all_contacts)\n        group_without_contacts = orm.get_groups_not_containing_contact(contact)\n        if len(group_without_contacts) == 0:\n            app.group.create(Group(name=\"name45\", header=\"header90\"))\n            group_without_contacts = orm.get_groups_not_containing_contact(contact)\n        group = random.choice(group_without_contacts)\n        app.contact.add_contact_to_group_by_id(contact.id, group.id, group.name)\n>       contacts_in_group = orm.get_contacts_in_group(group)\n\ntest/test_add_some_contact_to_group.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n<string>:2: in get_contacts_in_group\n    ???\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:552: in new_func\n    reraise(exc_type, exc, tb)\n../../.local/lib/python3.8/site-packages/pony/utils/utils.py:95: in reraise\n    try: raise exc.with_traceback(tb)\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:533: in new_func\n    result = func(*args, **kwargs)\nfixture/orm.py:75: in get_contacts_in_group\n    return self.convert_contacts_to_model(orm_group.contacts)\nfixture/orm.py:62: in convert_contacts_to_model\n    return list(map(convert, contacts))\nfixture/orm.py:55: in convert\n    return NewContact(id=str(contact.id), first_name=contact.firstname, middle_name=contact.middlename,\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:2288: in __get__\n    value = attr.get(obj)\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:2298: in get\n    val = vals[attr] if attr in vals else attr.load(obj)\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:2282: in load\n    else: obj._load_()\n../../.local/lib/python3.8/site-packages/pony/orm/core.py:4833: in _load_\n    if obj not in objects: throw(UnrepeatableReadError,\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nexc_type = <class 'pony.orm.core.UnrepeatableReadError'>, args = ('Phantom object ORMContact[59] disappeared',), kwargs = {}\n\n    def throw(exc_type, *args, **kwargs):\n        if isinstance(exc_type, Exception):\n            assert not args and not kwargs\n            exc = exc_type\n        else: exc = exc_type(*args, **kwargs)\n        exc.__cause__ = None\n        try:\n            if not (pony.MODE == 'INTERACTIVE' and options.CUT_TRACEBACK):\n>               raise exc\nE               pony.orm.core.UnrepeatableReadError: Phantom object ORMContact[59] disappeared\n\n../../.local/lib/python3.8/site-packages/pony/utils/utils.py:106: UnrepeatableReadError"}, "start": 1612531458973, "stop": 1612531465024, "uuid": "904b28a0-d92c-4d72-ac46-9be38ec836ee", "historyId": "ad7825a3edbca4e606616d9a19263ef7", "testCaseId": "df8e0e9c19f86ea5c292baaa4d439587", "fullName": "test.test_add_some_contact_to_group#test_add_contact_to_group", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_add_some_contact_to_group"}, {"name": "host", "value": "B570e2"}, {"name": "thread", "value": "9113-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_add_some_contact_to_group"}]}